Classes/Systems/LocalNotificationsCache.swift

```diff
@@ -19,7 +19,7 @@ final class LocalNotificationsCache {
 
     private let path: String
     private lazy var queue: FMDatabaseQueue = {
-        return FMDatabaseQueue(path: self.path)
+        return FMDatabaseQueue(path: self.path)!
```

Comment:
This now returns an optional; force unwrapping for now, as it seemingly worked in the past without error. :)